contracts/lib/MerkleProof.sol

```diff
@@ -1,21 +1,21 @@
-pragma solidity ^0.4.24;
+pragma solidity ^0.5.0;
 
 
 /*
  * @title MerkleProof
  * @dev Merkle proof verification based on
  * https://github.com/ameensol/merkle-tree-solidity/blob/master/src/MerkleProof.sol
  */
-library MerkleProofSha256 {
+library MerkleProof {
```

Comment:
I thought Zeppelin is providing the libraries as EVM packages?

Reply:
So we have a fork of zeppelin lib at the moment for sha3 support. EVM packages are instance based on a certain implementation and you need composition for leveraging them.  So in the case of a lib you deploy the contract.